### PR TITLE
Bugfix/lattice 2082 add members enrty processor

### PR DESCRIPTION
**Pull request by you-leee**

### CODE REVIEW

File: src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/CollectionStreamSerializer.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ *
+ */
+package com.kryptnostic.rhizome.hazelcast.serializers
+
+import com.hazelcast.nio.ObjectDataInput
+import com.hazelcast.nio.ObjectDataOutput
+import com.kryptnostic.rhizome.pods.hazelcast.SelfRegisteringStreamSerializer
+import java.io.IOException
+
+abstract class CollectionStreamSerializer<T : Collection<E>, E> protected constructor(private val clazz: Class<T>)
```
Comment (katherinebernstein) [inline src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/CollectionStreamSerializer.kt]: 🙌

File: src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/SetStreamSerializer.kt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * You can contact the owner of the copyright at support@openlattice.com
+ *
+ *
+ */
+package com.kryptnostic.rhizome.hazelcast.serializers
+
+import com.hazelcast.nio.ObjectDataInput
+import java.io.IOException
+
+abstract class SetStreamSerializer<T : MutableSet<E>, E> protected constructor(clazz: Class<T>)
```
Comment (katherinebernstein) [inline src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/SetStreamSerializer.kt]: 🙌

Comment (you-leee) [reply src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/SetStreamSerializer.kt]: 🙌

File: src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/CollectionStreamSerializer.kt (hunk 1 above)
Comment (you-leee) [reply src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/CollectionStreamSerializer.kt]: 🙌

**Pull request merged**